guidelines/enhancement_template.md

```diff
@@ -0,0 +1,258 @@
+---
```

Comment:
kube recently updated to a multi-file system.  Openshift hasn't followed yet.  I don't have strong opinions at smaller scales, just mentioning it.

Reply:
i put  the metadata in a separate file.